c-sharp/c-sharp-quiz.md

```diff
@@ -220,7 +220,7 @@ foreach(string employee in employeeQuery)
 - [ ] //\* Multiline
       / Single Line
 - [x] // Single Line
-      /_ Multiline _/
+      /* Multiline */
```

Comment:
it could be auto formatted with prettier
can you please also add answers inside backtick "```" to freeze the answers, thanks

Reply:
Did this in my new commit can you please check?